_posts/2020-12-02-spack-user-survey-2020.md

```diff
@@ -0,0 +1,918 @@
+the "University HPC center" and "public research lab" categories.
+
+## What country are you in?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_country.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_country.svg">
+  </a>
+</figure>
+
+Just under 2/3 of Spack users are in the United States, and nearly
+exactly 2/3 are from North America when our two Canadian respondents are
+included. 27% are from Europe, 5% from Asia, and there was one respondent
+each from the Middle East (Saudi Arabia) and South America (Argentina).
+Within ECP (which is a U.S. DOE project), the proportion is much higher
+-- nearly 97% are from the U.S.
+
+## What are your primary application areas?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_multi_bars_app_area.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_multi_bars_app_area.svg">
+  </a>
+</figure>
+
+These results are mostly as we expected -- most Spack users (~80%) are
+doing traditional HPC and simulation. In the broader community around 30%
+are doing computer science research, but within ECP around 50% are doing
+CS research. In ECP, AI and bioinformatics were noticeably less emphasized
+than in the broader Spack community. Interestingly, compiler testing was
+the 7th most popular application area outside ECP, but the 4th most
+popular inside ECP. Only one user reported using Spack for web
+applications.
+
+## How did you find out about Spack?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_how_find_out.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_how_find_out.svg">
+  </a>
+</figure>
+
+Both in and outside ECP, about half of Spack users hear about the tool
+via word of mouth. This result made us pretty happy -- we think it means
+that users are very willing to recommend Spack to friends. After word of
+mouth, 22% people heard of Spack because it was used at their site.
+Within ECP this was slightly more likely at 30%.
+
+24% of users heard about Spack from outreach activities: tutorials, BOF
+sessions, and presentations.
+
+
+## How long have you been using Spack?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_bars_how_long_using.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_bars_how_long_using.svg">
+  </a>
+</figure>
+
+Spack usage has been growing over time, and the number of users who join
+the community each year is increasing. Most of the overall community has
+been using Spack for less than two years. We also see this effect in the
+number of contributors to the project on GitHub. In 2018, after the
+project had been publicly available for 4 years, there were around 300
+contributors. 2 years later, there are nearly 700.
+
+Within ECP, the community is older -- most people started using Spack in
+ECP 2-3 years ago. There has been less adoption since then, but we
+attribute this to Spack's rapid adoption in ECP. Spack caught on quickly
```

Comment:
Slightly confusing. ECP doesn't have newer adoptions b/c Spack is already de facto, right?
```suggestion
ECP 2-3 years ago. There has been less new adoption since then, which we
attribute to Spack's initial rapid adoption in ECP. Spack caught on quickly
```